### DISCUSSION TITLE
Can I change the class location/namespace for Livewire/Forms?

### DISCUSSION THREAD

**Discussion by marchershey** [livewire/livewire#9079] [1. Help]
I've personally never liked having a separate livewire directory, so is it possible to have a custom class namespace/location for livewire forms, similar to component's `class_namespace`?

**Comment by joshhanley** [comment]
@marchershey yeah you can put them wherever you like and they should work fine 🙂 as long as your `use` statement at the top of your component points to the correct namespace, they should work happily.
